.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-  "liveServer.settings.port": 5501
+  "liveServer.settings.port": 5501,
+  "editor.formatOnSave": false,
```

Comment:
any particular reason you disabled `format on save`? It's disabled for everyone now ;)

Reply:
sorry forgot to undo it, had some weird formating happening every time I saved so "files changed" would be huge when often only a few line of code was added or removed

Reply:
no problem, could you make a new PR with the fix? :)